controllers/machine_helpers.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// getMachinesInCluster returns all of the Machine objects
+// that belong to the cluster with given namespace/name
+func getMachinesInCluster(ctx context.Context, c client.Client, namespace, name string) ([]*clusterv1.Machine, error) {
```

Comment:
I would like to see a test case for this function and perhaps a rename to `getActiveMachinesInCluster`. It could be read as "get all machines in a cluster" but the function is getting the non-deleted (is active the right word?) machines.

Regardless of a rename, this functionality should be reflected in the godoc.

Reply:
+1 to rename